doc/conf.py

```diff
@@ -134,7 +120,7 @@
 project = 'Matplotlib'
 copyright = ('2002 - 2012 John Hunter, Darren Dale, Eric Firing, '
              'Michael Droettboom and the Matplotlib development '
-             'team; 2012 - 2016 The Matplotlib development team')
+             'team; 2012 - 2017 The Matplotlib development team')
```

Comment:
I can imagine in 3 years someone would realize it's showing "2012 - NULL_VALUE" something